cfy_manager/components/postgresql_server/postgresql_server.py

```diff
@@ -124,8 +147,30 @@ def _write_new_hba_file(self, lines, enable_remote_connections):
             if not re.search(PG_HBA_LISTEN_ALL_REGEX_PATTERN,
                              '\n'.join(lines)) and enable_remote_connections:
                 f.write('host all all 0.0.0.0/0 trust\n')
+            if config[POSTGRESQL_SERVER][SSL_ENABLED] and not \
+                    re.search(PG_HBA_HOSTSSL_REGEX_PATTERN, '\n'.join(lines)):
+                f.write('hostssl all all 0.0.0.0/0 md5 clientcert=1')
+
         return temp_hba_path
 
+    def _configure_ssl(self):
+        """
+        Copy SSL certificates to postgres data directory.
+        Configuration for postgresql.conf is done in _write_new_pgconfig_file
```

Comment:
I would say "postgresql.conf and pg_hba.conf configurations are handled in the create configuration step"

Reply:
fixed